pkgs/by-name/_0/_010editor/package.nix

```diff
@@ -53,17 +53,23 @@ stdenv.mkDerivation (finalAttrs: {
         mkdir -p $out/Applications
         cp -R *.app $out/Applications
       '';
+
       linuxInstall = ''
         mkdir -p $out/opt && cp -ar source/* $out/opt
-
-        # Unset wrapped QT plugins since they're already included in the package,
-        # else the program crashes because of the conflict
-        makeWrapper $out/opt/010editor $out/bin/010editor \
-          --unset QT_PLUGIN_PATH
-
-        # Copy the icon and generated desktop file
-        install -D $out/opt/010_icon_128x128.png $out/share/icons/hicolor/128x128/apps/010.png
-        install -D $desktopItem/share/applications/* -t $out/share/applications/
+        mkdir -p $out/bin
+        cat > $out/bin/010editor <<EOF
+        #! /usr/bin/env bash
```

Comment:
In general, you shouldn't hardcode this like that. Instead, use `#!${stdenv.shell}`